ynh-dev

```diff
@@ -2,6 +2,12 @@
 
 # shellcheck disable=SC2155,SC2034,SC2164
 
+if (( $(lsb_release -rs) >= 12 )); then
+    YARN_CMD=yarnpkg
```

Comment:
On Debian 12+, the package [seems to be `yarnpkg`](https://packages.debian.org/bookworm/yarnpkg) and `yarn` seems to correspond to `cmdtest`.